numba/cuda/tests/cudasim/test_cudasim_issues.py

```diff
@@ -1,14 +1,49 @@
 import threading
 
 import numpy as np
+from numpy import float64
 
-from numba import cuda
+from numba import cuda, numba
 from numba.cuda.testing import SerialMixin, skip_unless_cudasim
 import numba.cuda.simulator as simulator
 import unittest
 
 
 class TestCudaSimIssues(SerialMixin, unittest.TestCase):
+    BACKYARD_TYPE = [('statue', float64),
+                     ('newspaper', float64, (6, ))]
+
+    GOOSE_TYPE = [('garden', float64, (12,)),
+                  ('town', float64, (42,)),
+                  ('backyard', BACKYARD_TYPE)]
+
+    GOOSE_NP_DTYPE = np.dtype(GOOSE_TYPE, align=True)
+
+    GOOSE_NB_DTYPE = numba.from_dtype(GOOSE_NP_DTYPE)
```

Comment:
Seems like these are only used in https://github.com/numba/numba/pull/5343/files#diff-7dbb9cebfbec424ce906ec5ad437908aR24 and so should go in the scope of that function?